### PR TITLE
v1 of the security copilot page

### PR DESCRIPTION
**Pull request by rr404**
The added page is : https://pr-566.d1to60jd2gb6y6.amplifyapp.com/u/cti_api/integration_securitycopilot

### CODE REVIEW

File: crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+CrowdSec Intelligence Plugin for Microsoft Security Copilot allows you to get advanced insights on a malicious IP activity.  
+As part of the core plugins of Security copilot its setup and usage are very straight forward.
+
+This documentation will lead you through an easy setup and lead you through some example usage and prompts.
+
+## Configure the plugin
+
+### Prerequisite: retrieve your API Key
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md:13]: We could just link to the getting started page of the CTI? or if you want to keep them here we can keep this

File: crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+The plugin is using our CTI API to provide information on over 70M attackers recently reported by CrowdSec's network.  
+You can create a trial key or retrieve your existing keys in the [console](https://app.crowdsec.net/) in the "Settings" > "CTI API Keys" section.  
+If you need more details check out the [CTI API Key - getting started section](/cti_api/getting_started)
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md:16]: Can you make a ref to the file itself `(/cti_api/getting_started.mdx)` as the compiler will auto link the actual route

Comment (LaurenceJJones) [reply crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md:16]: As using it without leads to a 404 https://pr-566.d1to60jd2gb6y6.amplifyapp.com/cti_api/getting_started

Comment (rr404) [reply crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md:16]:
```suggestion
If you need more details check out the [CTI API Key - getting started section](/cti_api/getting_started.mdx)
```

**Review by mazzma12 — APPROVED** [review]
a few grammar mistake but better merge it

File: crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Configure the plugin
+
+### Prerequisite: retrieve your API Key
+The plugin is using our CTI API to provide information on over 70M attackers recently reported by CrowdSec's network.  
+You can create a trial key or retrieve your existing keys in the [console](https://app.crowdsec.net/) in the "Settings" > "CTI API Keys" section.  
+If you need more details check out the [CTI API Key - getting started section](/cti_api/getting_started.mdx)
+
+### Activate and setup the plugin
+This consist of 3 easy step: browse plugins, select "CrowdSec Threat Intelligence" plugin, paste API Key in setting
```
Comment (rr404) [inline crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md:19]:
```suggestion
This consists of 3 easy steps: browse plugins, select "CrowdSec Threat Intelligence" plugin, paste API Key in settings
```

**Review by AlteredCoder — APPROVED** [review]

**Pull request merged**